src/index.json

```diff
@@ -3207,8 +3207,8 @@
         ],
         "arcappliance": [
             {
-                "downloadUrl": "https://arcplatformcliextprod.blob.core.windows.net/arcappliance/arcappliance-0.1.1-py2.py3-none-any.whl",
-                "filename": "arcappliance-0.1.1-py2.py3-none-any.whl",
+                "downloadUrl": "https://arcplatformcliextprod.blob.core.windows.net/arcappliance/arcappliance-0.1.2-py2.py3-none-any.whl",
```

Comment:
Can you remove `py2` in the wheel name by removing the following settings in `setup.cfg` for the next version?
```
[bdist_wheels]
universal=1
```